pkg/virt-api/rest/subresource_test.go

```diff
@@ -1411,54 +1139,48 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 	})
 
 	Context("Subresource api - MigrateVMRequestHandler", func() {
-		It("should fail if VirtualMachine not exists", func(done Done) {
-			request.PathParameters()["name"] = "testvm"
-			request.PathParameters()["namespace"] = "default"
+		table.DescribeTable("should fail if VirtualMachine not exists", func(migrateOptions *v1.MigrateOptions) {
 
-			server.AppendHandlers(
-				ghttp.CombineHandlers(
-					ghttp.VerifyRequest("GET", "/apis/kubevirt.io/v1alpha3/namespaces/default/virtualmachines/testvm"),
-					ghttp.RespondWithJSONEncoded(http.StatusNotFound, nil),
-				),
-			)
+			request.PathParameters()["name"] = testVMName
+			request.PathParameters()["namespace"] = k8smetav1.NamespaceDefault
+
+			bytesRepresentation, _ := json.Marshal(migrateOptions)
+			request.Request.Body = io.NopCloser(bytes.NewReader(bytesRepresentation))
 
+			vmClient.EXPECT().Get(testVMName, &k8smetav1.GetOptions{}).Return(nil, errors.NewNotFound(v1.Resource("virtualmachine"), testVMName))
 			app.MigrateVMRequestHandler(request, response)
 
 			ExpectStatusErrorWithCode(recorder, http.StatusNotFound)
-			close(done)
-		}, 5)
+		},
+			table.Entry("", &v1.MigrateOptions{}),
```

Comment:
Description is left empty here. I guess something like `"In a real run"` or `"without dry-run option"` will do.

IMHO It's better to not keep these empty for two reasons, first being that it's clearer, and secondly since if another Entry would show up with an empty description it would create a mess.

P.S. - you have more of Entries with no description in this file, I'm not going to comment the same for every one but the same thing applies to all :)

Reply:
Thanks so much. Actually you're totally right. I will fix that :slightly_smiling_face: